npm-cli.adoc

```diff
@@ -0,0 +1,133 @@
+
+== Basic Usage
+
+`-v, --version`:: Prints the current version of npm you are using
+`<command> -h, --help`:: quick help on the specified command
+`-l`:: display full usage info
+
+`npx [command]`:: Run an arbitrary command from an npm module (either one installed locally, or fetched remotely)
+
+== Create a new Node.js application
+
+`npm init [--yes|-y|--scope]`:: Initialize a new Node.js Project,  Create a package.json file
+
+[cols="25,75"]
+|===
+|`--yes, -y`
+| Skip the init questionnaire
+
+|`--scope`
+| Create a scoped application/module
+|===
+
+== Install Node Modules
+
+`npm install <module_name>`:: Installs the specifed module as a Dependency
+
+[cols="25,75"]
+|===
+|`--save-dev`
+| Installs the specifed module as a Development Dependecy
+
+|`-g`
+| Installs the module globally
+|===
+
+`npm install [--production]`:: Installs the modules specified in the package.json.  The `--production` flag will not install development dependencies
+
+`npm ci`:: similar to `npm install`, but for use in automated environments such as test platforms, continuous integration, and deployment
+
+`npm uninstall <module>`:: remove a module
+
+`npm outdated`:: Check for outdated modules
+
+`npm update`:: Update modules to the latest version, respecting semver.
+
+`npm ls`:: List the installed modules
+
+`npm prune`:: Remove any extraneous modules
+
+`npm link`:: Symlink a package folder
+
+== NPM Scripts
+
+`npm start`:: Run a predefined command specified in the `start` property of a package's `scripts` object.
+
+`npm run [script]::` Run a predefined command specified in the `[script]` property of package's `scripts` object
+
+== Publish a Node Module
+
+`npm whoami`:: Display the npm username
+
+`npm login`:: Login to the registry
+
+`npm version [options]`:: Bump the version of a module
+
+[cols="25,75"]
+|===
+|`<newversion>`
+| Bumps the version to the `<newversion>`.  ex: `npm version 1.1.0`
+
+|`major`
+| Bumps the major version by 1. ex: `npm version major` output: `v2.1.0`
+
+|`minor`
+| Bumps the minor verison by 1. ex: `npm version minor` output `v1.2.0`
+
+|`patch`
+| Bumps the patch version by 1. ex: `npm version patch` output: `v1.1.1`
+
+|`premajor`
+| Bumps the major version by 1, but as a prerelease. ex: `npm verison premajor` output: `v2.1.0-0`
+
+|`preminor`
+| Bumps the minor version by 1, but as a prerelease. ex: `npm verison preminor` output: `v1.2.0-0`
+
+|`prepatch`
+| Bumps the patch version by 1, but as a prerelease. ex: `npm verison prepatch` output: `v1.1.1-0`
+
+|`prerelease`
+| Bumps the prerelease version.  ex: `npm version prerelease` output: `1.1.0-1`
+
+|`--preid=<prerelease-id>`
+| Use a custom prerelease id tag.  ex: `npm version prelease --preid=beta` example output: `1.1.0-beta.0`
+
+|`from-git`
+| Use the latest git tag as the version
+|===
+
+`npm shrinkwrap`:: Lock down dependency versions for publication
+
+`npm publish`:: Publish a module
```

Comment:
May be useful to mention the `--dry-run` option to see what would be published without actually publishing.

Reply:
Yea,  good call on that.  i'll add it